### ISSUE TITLE
WAVE CSTF process for approving supplemental data profiles

### ISSUE THREAD

**Issue by johnsim** [cta-wave/content-specification-task-force#23]
It was agreed in the July 25, 2018 cstf special session that we need to add "... a new issue in GitHub to establish a process for approving supplemental data profiles (SDP) in the future."

The approval process for media profiles (MP) has been documented and agreed upon in the WAVE CSTF. There are questions that need resolving, such as:

- Are the requirements for a SDP distinct from those of an MP (section 2.2, of [WAVE MP]), or is it simply a matter of updating the language to mention both MP and SDP?

- A SDP is applied to MPs, and a client which only understands the underlying MP and not the SDP should perform normally. This is a device playback consideration, but is it also a content spec consideration?

- An SDP (SEI message) does have MSE requirements, which should be captured in the binding specification. Is there generic language we can place in WAVE regarding this, or must it be captured individually in each SDP CMAF binding spec?

======================================

"Special Meeting on Dolby Vision ‘cdv1’ Media Profile  Draft Meeting Report", https://standards.cta.tech/apps/org/workgroup/cstf/download.php/23750/Meeting%20Notes%20WAVE%20Content%20Spec%20TF%20Meeting_20180725.pdf 

[WAVE MP] "WAVE Media Profile Approval Process", https://standards.cta.tech/apps/org/workgroup/cstf/download.php/22529/cstf-00035-v000-WAVE%20Media%20Profile%20Approval%20Process.docx

**Comment by irajs** [comment]
On the 2nd bullet, the content spec currently does not distinguish between MPD and SDPs.  Section 3.3 introduces media profiles. Suggest the following:
1. add the paragraph to introduce SDPs and how a SDP is used on the top of a (relevant) media profile.
2.  modify figure 5, or add a note that Media Profile N might be Media Profile N + SDP M, when applicable.

Also the WAVE baseline splice constraint profile applies to all media profiles. We need to clarify whether the same constraints applies to SDPs.

**Comment by johnsim** [comment]
Agreed in the October 24, 2018 conference call - we will postpone SDP process until after publication of AMD 1 - and proceed with the SDP approval process to be completed in time for our Spring 2019 release.
